2018/python/lar_constraints.py

```diff
@@ -130,8 +131,10 @@ def v628_1_const(self, row):
 			   unless an ethnicity is provided in Ethnicity of Applicant or Borrower: Free Form Text Field for Other
 			   Hispanic or Latino."""
 		eth_enums = ["1", "11", "12", "13", "14", "2", "3", "4"]
-		if row["app_eth_1"] =="" and row["app_eth_free"] =="":
+		if row["app_eth_free"] == "" and row["app_eth_1"] not in eth_enums:
 			row["app_eth_1"] = random.choice(eth_enums)
+		if row["app_eth_free"] != "" and row["app_eth_1"] in eth_enums:
```

Comment:
Why set the free text to blank under these conditions? Code 14 is in the eth_enums list and is a valid combination with free text. 

Setting the free text field to blank will be incompatible if code 14 is the result of the random function.

Reply:
I wanted to observe the language in the FIG that includes code 14 as part of the check. I think I can add another check that allows for a blank value with Code 14.

Reply:
Can you please rephrase? This PR title is about v630_2, which makes sense for both years. 

Can you add some text to the PR description and the code so that I can understand why these changes are being made to V628? Can you also explain the logic in that text?

Thank you

Reply:
This comment was market resolved, but I don't see any additions to the PR text explaining the scope of the changes. Can you please add that?

Reply:
Sure - apologies, I think I needed to refresh the screen before marking this comment resolved. I can rephrase this.